lib/api-requestor.js

```diff
@@ -71,7 +71,7 @@ module.exports = class ApiRequestor {
       if (this._chargehound.options.protocol === 'https://') {
         req = https.request(reqOpts).setTimeout(this.getTimeout())
         connectEv = 'secureConnect'
-      } else {
+      } else if ((this._chargehound.options.protocol === 'http://') && (this._chargehound.options.host === 'localhost')) {
```

Comment:
python dont have else if. please change it to elif

Reply:
I agree with deepak, can we check current values of protocol  for all environment (local, stage, prod) and use it accordingly here instead of relying on protocol value